resources/lib/utils.py

```diff
@@ -136,7 +116,7 @@ def url_to_program(url):
     elif url.startswith('//www.vrt.be/vrtnu/a-z/'):
         # medium programUrl or targetUrl
         program = url.split('/')[5]
-    elif url.startswith('/vrtnu/a-z/'):
+    elif url.startswith('/vrtnu/a-z/') or url.startswith('/vrtmax/a-z/'):
```

Comment:
Used or instead of dropping 'vrtnu' since this would be a BC break for users with local HTTP cache.